addon/configuration.js

```diff
@@ -47,7 +47,7 @@ export default {
     route is not authenticated.
 
     @property authenticationRoute
-    @deprecated 'deprecated in favor of overriding the authenticationRoute method on the authenticatedRouteMixin'
+    @deprecated AuthenticatedRouteMixin/authenticationRoute:property
```

Comment:
YUIDoc doesn't allow crosslinks or any formatting in the deprecation message. So instead of providing an actual text here we simply only provide the reference to the things that this elements has been deprecated in favor of so that we can link to that in the docs.